app/models/user/academics/diploma.rb

```diff
@@ -18,6 +25,10 @@ def as_json(options = {})
     }
   end
 
+  def as_json(options = {})
+    get_feed
+  end
+
```

Comment:
I would much prefer that `get_feed`/`get_feed_internal` use `as_json` than the other way around